app/src/main/res/values/strings.xml

```diff
@@ -129,4 +133,10 @@
         <item quantity="one">%d new message</item>
         <item quantity="other">%d new messages</item>
     </plurals>
+    <string name="maximum_time_limit_error">Maximum time limit exceeded</string>
+    <string name="ok">OK</string>
+    <string name="cancel">Cancel</string>
+    <string name="server_time_error_response">The time limit for editing this message has past</string>
+    <string name="is_maximum_content_edit_limit_saved">IS_MAXIMUM_CONTENT_EDIT_LIMIT_SAVED</string>
```

Comment:
Strings.xml mainly used to contain user displayed text move this constant somewhere else!